### DISCUSSION TITLE
Cycle Lore

### DISCUSSION THREAD

**Discussion by cursorweb** [cursorweb/Cycle-Bot-Game#15] [General]
The next thing I'm focusing on for Cycle is its *lore*

That is
- The creation of ego-coins
- Prestige
- Crafting

Any ideas?

**Comment by PotentialStyx** [comment]
ego-coins came from the mystical being that is coder100.

crafting came from people with not enough coins to buy items, so instead they learned how to make the items themself's.

prestige is when mystical coder100 senses someone getting too close to his ego-coin count so he gives them the option to restart with a boost, which seems good in the beginning but is really a trick.

**Reply by PotentialStyx** [reply]
obtw I could help a bit 👉👈

**Reply by cursorweb** [reply]
yay thanks

and sure, would you like to be a collaborator?

**Comment by thewaydev** [comment]
First people could only become more powerful, until a wizzard named thewaydu found a way to, go back....

**Reply by cursorweb** [reply]
lol
